budget_proj/budget_app/views.py

```diff
@@ -60,6 +60,27 @@ class gives you a convenient way to declare views quickly when you only
     queryset = find_ocrb_data()
     serializer_class = serializers.OcrbSerializer
 
+class OcrbList(generics.ListCreateAPIView):
+    """
+    Class based view that inherits from the generics class and pulls from AWS
+    """
+    queryset = models.OCRB.objects.all()
+    serializer_class = serializers.OcrbSerializer
+    def get(self, request, *args, **kwargs):
+        #check for query params
+        #eg: http://127.0.0.1:8000/ocrb-prod/?amount=0
+        k = request.GET.keys()
+        filter_dict = {}
+        if(k):
+            for key, value in request.GET.items():
+                filter_dict[key] = value
```

Comment:
I like your use of a dictionary of query parameters better than the long list of if-statements that I implemented in the `FindOperatingAndCapitalRequirements` class.